### ISSUE TITLE
How to include files with custom extensions?

### ISSUE THREAD

**Issue by bersbersbers** [microsoft/pyright#7778]
**Describe the bug**
I seem to be unable to include custom extensions into `pyright`:

**Code or Screenshots**
`bug.spec`
```python
a: int = "1"
```

`pyproject.toml`
```toml
[tool.pyright]
include = ["*.spec", "**/*.spec", "**/*.*"]
ignore = []
```

`pyright` passes:
> 0 errors, 0 warnings, 0 informations 

`pyright bug.spec` works;
```
c:\Code\bug\bug.spec
  c:\Code\bug\bug.spec:1:10 - error: Expression of type "Literal['1']" is incompatible with declared type "int"
    "Literal['1']" is incompatible with "int" (reportAssignmentType)
1 error, 0 warnings, 0 informations
```

**VS Code extension or command-line**
```
(.venv) C:\Code\bug>pyright --version
pyright 1.1.360
```

**Comment by erictraut** [comment]
Pyright's normal file search mechanism filters based on standard python file extensions (".py" and ".pyi").

If you want to check files with non-standard extensions, you need to pass the names specifically on the command line.

**Comment by bersbersbers** [comment]
Thanks, understood. Would it be useful to document that? Because the current docs say

> **include** [array of paths, optional]: Paths of directories **or files** that should be included

with wildcards also being mentioned (https://microsoft.github.io/pyright/#/configuration?id=main-configuration-options), and to me, this implies custom extensions should be working.
